resources/language/resource.language.hu_hu/strings.po

```diff
@@ -968,16 +968,16 @@ msgstr "A háttér folyamat nem indult el a következő hiba miatt:\r\n{}"
 
 msgctxt "#30241"
 msgid "Open Connect CDN (the lower index is better)"
-msgstr "Open Connect CDN (alacsonyabb index jobb)"
+msgstr "Nyílt CDN csatlakozás (alacsonyabb index jobb)"
```

Comment:
original was right, "Open Connect" is a brand do not to translate

Reply:
restored